packages/docusaurus-utils/src/cliUtils.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import prompts from 'prompts';
+import logger from '@docusaurus/logger';
+
+export type LanguagesOptions = {
+  javascript?: boolean;
+  typescript?: boolean;
+};
+
+export async function getLanguage(
+  languages: LanguagesOptions,
+  noTsVersionAvailable?: boolean,
+): Promise<LanguagesOptions> {
+  if (languages.typescript || languages.javascript) {
+    return languages;
+  }
+  if (noTsVersionAvailable) {
+    return {javascript: true};
+  }
```

Comment:
I'm not sure it's a good thing to have this thing here, will try to see where to move it